crates/task-impls/src/quorum_vote.rs

```diff
@@ -350,6 +350,7 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> QuorumVoteTaskState<TYPES, I
 
     /// Handle a vote dependent event received on the event stream
     #[instrument(skip_all, fields(id = self.id, latest_voted_view = *self.latest_voted_view), name = "Quorum vote handle", level = "error")]
+    #[allow(unused_variables)]
```

Comment:
Nit: Why do we need this? Looks like all variables are used.

Reply:
I think it has to do with the function not being called? maybe it can be removed now but clippy was complaining